packages/liveblocks-react/src/comments/CommentsRoom.ts

```diff
@@ -99,6 +100,10 @@ export type RoomThreads<TThreadMetadata extends BaseMetadata> =
       error?: never;
     };
 
+export type RoomThreadsSuspense<TThreadMetadata extends BaseMetadata> = Resolve<
+  Extract<RoomThreads<TThreadMetadata>, { isLoading: false; error?: never }>
```

Comment:
This construct is unnecessarily hard to read and might be hard to maintain. `RoomThreads` is a union of three members, of which you're excluding two here. Why not name those members explicitly, and just refer to [the one member](https://github.com/liveblocks/liveblocks/blob/168610b68f879da0f977864fa52b0d56c9221c88/packages/liveblocks-react/src/comments/CommentsRoom.ts#L87-L91) you're interested in here by a named type? I think it will make this much simpler to follow. You can even drop the `Resolve` helper too, and we can use "jump to definition".

Reply:
Agree! Changed in https://github.com/liveblocks/liveblocks/pull/1156/commits/c03ef95d1608fd317ddb61d92d494d03e8e7bec7.